src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,251 @@
+package Duke;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import Duke.Exceptions.*;
+import Duke.Task.*;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class Duke {
+
+    private static ArrayList<Task> tasks = new ArrayList<>();
+    private static int totalTasks = 0;
+
+    private static final String home = System.getProperty("user.home");
+    private static final String dirPath = home + File.separator + "Documents" + File.separator + "CS2113T";
+    private static final String filePath = home + File.separator + "Documents" + File.separator + "CS2113T"
+            + File.separator + "tasks.txt";
+
+    private static void loadFile() {
+        if(!new File(dirPath).exists()) {
```

Comment:
There should a space between `if` and `(` so it should be `if (!new File....)) {`

Reply:
The same applies to all other instances of your `if` statements.